From 4ae41b01ba2dfd05e8eae0adac6dc3d54c461117 Mon Sep 17 00:00:00 2001
From: songyanting <songyanting@huawei.com>
Date: Mon, 24 Jan 2022 11:08:44 +0800
Subject: [PATCH] [Huawei]containerd:optimize cgo compile options

offering:EulerOS Server
Type:bugfix
CVE:
DTS/AR:
reason:optimize cgo compile options

Signed-off-by: songyanting songyanting@huawei.com
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index a7d0888..49a90e6 100644
--- a/Makefile
+++ b/Makefile
@@ -172,8 +172,8 @@ bin/%: cmd/% FORCE
 	mkdir -p $(BEP_DIR)
 	@echo "$(WHALE) $@${BINARY_SUFFIX}"
 	CGO_ENABLED=1 \
-	CGO_CFLAGS="-fstack-protector-strong -fPIE" \
-	CGO_CPPFLAGS="-fstack-protector-strong -fPIE" \
+	CGO_CFLAGS="-fstack-protector-strong" \
+	CGO_CPPFLAGS="-fstack-protector-strong" \
 	CGO_LDFLAGS_ALLOW='-Wl,-z,relro,-z,now' \
 	CGO_LDFLAGS="-Wl,-z,relro,-z,now -Wl,-z,noexecstack" \
 	go build ${GO_GCFLAGS} ${GO_BUILD_FLAGS} -o $@${BINARY_SUFFIX} ${GO_LDFLAGS} ${GO_TAGS}  ./$<
-- 
2.23.0

